src/libstd/f32.rs

```diff
@@ -1080,6 +1080,27 @@ impl f32 {
         0.5 * ((2.0 * self) / (1.0 - self)).ln_1p()
     }
 
+    /// Returns max if self is greater than max, and min if self is less than min.
+    /// Otherwise this returns self.  Panics if min > max, min equals NaN, or max equals NaN.
```

Comment:
Should this be "is `NaN`" and not "equals `NaN`"? Nothing is equal (as in `==`) to `NaN`, not even `NaN` (if I recall correctly).